snap/implicit.go

```diff
@@ -27,41 +27,50 @@ import (
 	"github.com/snapcore/snapd/release"
 )
 
-var implicitSlots = []string{
-	"alsa",
-	"bluetooth-control",
-	"camera",
-	"dcdbas-control",
-	"docker-support",
-	"firewall-control",
-	"fuse-support",
-	"hardware-observe",
-	"home",
-	"kernel-module-control",
-	"locale-control",
-	"log-observe",
-	"lxd-support",
-	"mount-observe",
-	"network",
-	"network-bind",
-	"network-control",
-	"network-observe",
-	"network-setup-observe",
-	"opengl",
-	"ppp",
-	"process-control",
-	"raw-usb",
-	"removable-media",
-	"shutdown",
-	"snapd-control",
-	"system-observe",
-	"system-trace",
-	"time-control",
-	"timeserver-control",
-	"timezone-control",
-	"tpm",
+func initImplicitSlots() []string {
```

Comment:
No need to change here. See below.. we're already building up the final list inside `AddImplicitSlots`. Let's just add the `fuse-support` there conditionally.